extensions/common/json_util.cc

```diff
@@ -0,0 +1,114 @@
+/* Copyright 2020 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "extensions/common/json_util.h"
+
+#include "absl/strings/numbers.h"
+
+namespace Wasm {
+namespace Common {
+
+::nlohmann::json JsonParse(absl::string_view str) {
+  return ::nlohmann::json::parse(str, nullptr, false);
```

Comment:
I guess we will need to extend this to support json marshaling in the future, will that be easy?

Reply:
Can you elaborate? There is `json::dump()` that prints out JSONs

Reply:
Like marshaling a struct to json or build a json string for outbound http request (OPA for example). Not sure if the library provides handy functions for this.

Reply:
Building JSON is very easy with the library. E.g.:
```
json j2 = {
  {"pi", 3.141},
  {"happy", true},
  {"name", "Niels"},
  {"nothing", nullptr},
  {"answer", {
    {"everything", 42}
  }},
  {"list", {1, 0, 2}},
  {"object", {
    {"currency", "USD"},
    {"value", 42.99}
  }}
};
```